src/advec_cell.cpp

```diff
@@ -129,7 +126,6 @@ void advec_cell_kernel(
 						sigma3 = (1.0 + sigmat) * (vertexdx[j] / vertexdx[dif]);
 						sigma4 = 2.0 - sigmat;
 
-						sigma = sigmat;
```

Comment:
We should keep this for similar reasons to above.

Reply:
Will keep, although it was created, assigned, and never used again and it doesn't escape the kernel it any way.